openapi_python_client/templates/model.pyi

```diff
@@ -66,8 +83,37 @@ class {{ model.reference.class_name }}:
     {% endif %}
 
 {% endfor %}
-        return {{ model.reference.class_name }}(
+        {{model.reference.module_name}} = {{ model.reference.class_name }}(
 {% for property in model.required_properties + model.optional_properties %}
             {{ property.python_name }}={{ property.python_name }},
 {% endfor %}
         )
+
+        {% if model.additional_properties %}
+        {{model.reference.module_name}}._additional_properties.update(d)
+        {% endif %}
+        return {{model.reference.module_name}}
+
+    {% if model.additional_properties %}
+
+    @property
+    def additional_properties(self) -> Dict[str, {{ additional_property_type }}]:
+        return self._additional_properties
```

Comment:
What's the benefit of having this internal w/ `@property` vs just declaring it as `additional_properties` originally?